proseco/guide.py

```diff
@@ -49,21 +56,39 @@ def get_guide_catalog(obsid=0, **kwargs):
     guides.set_attrs_from_kwargs(obsid=obsid, **kwargs)
     guides.set_stars()
 
+    # Process monitor window requests, converting them into fake stars that
+    # are added to the include_ids list.
+    guides.process_monitors_pre()
+
     # Do a first cut of the stars to get a set of reasonable candidates
     guides.cand_guides = guides.get_initial_guide_candidates()
 
+    # Process guide-from-monitor requests by finding corresponding star in
+    # cand_guides and adding to the include_ids list.
+    # guides.process_monitors_pre2()
```

Comment:
?

Reply:
Oops, that's a leftover. The previous comment mentioning fake stars is also a leftover. Will fix.